modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewMouseInputTest.java

```diff
@@ -554,6 +554,7 @@ private void test_rt_38306(boolean selectTwoRows) {
         TableView.TableViewSelectionModel sm = table.getSelectionModel();
         sm.setCellSelectionEnabled(true);
         sm.setSelectionMode(SelectionMode.MULTIPLE);
+        sm.clearSelection();
 
```

Comment:
wondering: why did you add the clearSelection?

Reply:
it was an attempt to ensure a clear initial state, but you are right - it is unnecessary.